pkg/controller/knativeserving/openshift/openshift.go

```diff
@@ -32,6 +32,11 @@ const (
 
 	// The secret in which the tls certificate for the autoscaler will be written.
 	autoscalerTlsSecretName = "autoscaler-adapter-tls"
+	// knativeServingInstalledNamespace is the ns where knative serving operator has been installed
+	knativeServingInstalledNamespace = "NAMESPACE"
+	// service monitor created successfully when monitoringLabel added to namespace
+	monitoringLabel = "openshift.io/cluster-monitoring"
+	rolePath        = "deploy/resources/monitoring/role_service_monitor.yaml"
```

Comment:
Now, this rolePath should be moved into `installServiceMonitor()`.

Reply:
Done